inline/README.md

```
@@ -0,0 +1,16 @@
+# ⭐ optimize
+
+
+
+On rare occasions, `yield*`` syntax can cause a performance degradation, for example when there are many, many levels of recursion.
+
+The idea is that we can roll up any number of levels of recursion into a single yield point by transforming:
+```js
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)